src/main/java/com/backbase/billpay/fiserv/ebills/EbillsController.java

```diff
@@ -36,14 +39,14 @@ public void deleteBillPayEbills(String payeeId, String subscriberId, HttpServlet
     }
     
     @Override
-    public BillPayEbillsGetResponseBody getBillPayEbills(String payeeId, String subscriberId, String status, Date startDate,
-                    Date endDate, Integer from, Integer size, String orderBy, String direction,
+    public BillPayEbillsGetResponseBody getBillPayEbills(String payeeId, String subscriberId, String status, LocalDate startDate,
+                    LocalDate endDate, Integer from, Integer size, String orderBy, String direction,
                     HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse) {
         logger.debug("Retrieving eBills for payee with id: {}, status: {}, startDate: {}, endDate: {}, from: {}, "
                         + "size: {}, orderBy: {}, direction: {}", payeeId, status, startDate, endDate, from, size,
                         orderBy, direction);
         Header header = fiservUtils.createHeader(httpServletRequest, subscriberId);
-        BillPayEbillsGetResponseBody response = ebillsService.getEbills(header, payeeId, status, startDate, endDate,
+        BillPayEbillsGetResponseBody response = ebillsService.getEbills(header, payeeId, status, fromLocalDate(startDate), fromLocalDate(endDate),
```

Comment:
It would be best to keep all of the mapping logic within the same place, so converting the service method to take LocalDate and then changing to Date where required when calling Payveris instead.